installers/metrics/ansible/roles/pgo-metrics/templates/alertmanager-deployment.json.j2

```diff
@@ -0,0 +1,95 @@
+{
+    "apiVersion": "apps/v1",
+    "kind": "Deployment",
+    "metadata": {
+        "name": "crunchy-alertmanager",
+        "labels": {
+            "vendor": "crunchydata"
+        }
+    },
+    "spec": {
+        "replicas": 1,
+        "selector": {
+          "matchLabels": {
+            "name": "{{ alertmanager_service_name }}",
+            "vendor": "crunchydata"
+          }
+        },
+        "template": {
+            "metadata": {
+                "labels": {
+                    "name": "{{ alertmanager_service_name }}",
+                    "vendor": "crunchydata"
+                }
+            },
+            "spec": {
+                "securityContext": {
+{% if (alertmanager_supplemental_groups | default('')) != '' %}
+                    "supplementalGroups": [{{ alertmanager_supplemental_groups }}]
+{% endif %}
+{% if not (disable_fsgroup | default(false) | bool) %}
+                    {% if (alertmanager_supplemental_groups | default('')) != '' %},{% endif -%}
+                    "fsGroup": 26,
+                    "runAsUser": 2
+{% endif %}
+                },
+                "serviceAccountName": "alertmanager",
+                "containers": [
+                    {
+                        "name": "alertmanager",
+                        "image": "{{ alertmanager_image_prefix }}/{{ alertmanager_image_name }}:{{ alertmanager_image_tag }}",
+                        "args": [
+                            "--config.file=/etc/alertmanager/alertmanager.yml",
+                            "--storage.path=/alertmanager",
+                            "--log.level={{ alertmanager_log_level }}"
+                        ],
+                        "ports": [
+                            {
+                                "containerPort": {{ alertmanager_port }},
+                                "protocol": "TCP"
+                            }
+                        ],
+                        "readinessProbe": {
+                            "tcpSocket": {
+                                "port": {{ alertmanager_port }}
+                            },
+                            "initialDelaySeconds": 20,
+                            "periodSeconds": 10
+                        },
+                        "livenessProbe": {
+                            "tcpSocket": {
+                                "port": {{ alertmanager_port }}
+                            },
+                            "initialDelaySeconds": 25,
+                            "periodSeconds": 20
+                        },
```

Comment:
Alertmanager has HTTP endpoints for these kinds of probes. https://prometheus.io/docs/alerting/latest/management_api/

```suggestion
                        "readinessProbe": {
                            "httpGet": {
                                "path": "/-/ready",
                                "port": {{ alertmanager_port }}
                            },
                            "periodSeconds": 10
                        },
                        "livenessProbe": {
                            "httpGet": {
                                "path": "/-/healthy",
                                "port": {{ alertmanager_port }}
                            },
                            "initialDelaySeconds": 25,
                            "periodSeconds": 20
                        },
```